scripts/release.kts

```diff
@@ -61,21 +51,18 @@ File(path)
         val module = folder.name
 
         val file: String
-        val repositoryUrl: String
 
         val androidFile = folder.listFiles { it -> it.name.contains("release") && it.extension == "aar" }.firstOrNull()
         if (androidFile != null) {
             file = androidFile.path
-            repositoryUrl = androidRepositoryUrl
         } else {
             file = "$path/$module.jar"
-            repositoryUrl = javaRepositoryUrl
         }
         val javadocFile = "$path/$module-javadoc.jar"
         val sourcesFile = "$path/$module-sources.jar"
         val pomFile = "$path/pom-default.xml"
 
-        val command = "./mvnw deploy:deploy-file -Dfile=$file -Dfiles=$javadocFile,$sourcesFile -Dclassifiers=sources,javadoc -Dtypes=jar,jar -DpomFile=$pomFile -DrepositoryId=$repositoryId -Durl=$repositoryUrl\\;publish\\=$publish --settings $settingsPath"
+        val command = "./mvnw deploy:deploy-file -Dfile=$file -Dfiles=$javadocFile,$sourcesFile -Dclassifiers=sources,javadoc -Dtypes=jar,jar -DpomFile=$pomFile -DrepositoryId=$repositoryId -Durl=$repositoryUrl --settings $settingsPath"
```

Comment:
do you know if this actually closes the nexus repository automatically or still requires UI clicks?

Reply:
`-Dtypes=jar,jar` should be dynamic based on artifacts type? `aar` for Android

Reply:
AFAIK still requires UI click

Reply:
better to double-check then, some plugins (via Gradle) do have an automatic release eg https://github.com/vanniktech/gradle-maven-publish-plugin#releasing

there are also CLI tools that do the releasing automatically eg https://github.com/saket/startship

the current way would still keep the release process manual, which is fine, because this PR is about moving away from bintray, but I'm not sure if we should go this way if implementing Craft would require ripping this off anyway.

let's not block this PR because of that but lets keep this on our minds :)

Reply:
> -Dtypes=jar,jar should be dynamic based on artifacts type? aar for Android
Isnt this for javadocs and sources files?

Reply:
if this is only for javadocs and sourcefile yes, or does it also publish the jar/aar file? that's what I understand from:

`-Dfile=$file -Dfiles=$javadocFile,$sourcesFile`